CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+8.2.0...
+
+### Breaking changes
+
+The "labels to fields" transformer was changed so that label values are ordered after the value and multiple frames are not
```

Comment:
Should we mention that if anyone still want to keep the auto-merge they can just add a merge transformation after this transformation.